Framework/Kernel/src/VectorHelper.cpp

```diff
@@ -548,17 +548,16 @@ bool isConstantValue(const std::vector<double> &arra) {
 /** Take a string of comma or space-separated values, and splits it into
  * a vector of doubles.
  * @param listString :: a string like "0.0 1.2" or "2.4, 5.67, 88"
- * @return a vector of doubles
+ * @param separator :: Specify the string in which to split into vector. Default space and comma: " ,"
```

Comment:
We should be clear here that it's using `is_any_of`. I.e. Any of the characters used in the string will be used as delimiters.

`"0,1 2, 3"` would still produce a `[0, 1, 2, 3]` vector with the `separator` set to ` ,`. 

Maybe renaming `separator` to `separators` would be more clear? 

It would also be good to add a test for this behaviour.